src/components/Header/TopNav/outerItemStates.js

```diff
@@ -1,61 +1,85 @@
 import { css } from 'styled-components'
 
-export const defaultLightStyles = css`
+const defaultLight = css`
   background: ${props => props.theme.colors.white};
   color: ${props => props.theme.colors.text};
   opacity: 1;
 `
 
-export const hoverLightStyles = css`
+const hoverLight = css`
   background: ${props => props.theme.colors.greyBG};
   color: ${props => props.theme.colors.text};
   opacity: 1;
```

Comment:
It seems that since the `opacity: 1` is on the default state, some of these `opacity: 1` rule can be remove, or am I wrong?